fri/src/prover/mod.rs

```diff
@@ -171,7 +181,7 @@ where
 
         // reduce the degree by folding_factor at each iteration until the remaining polynomial
         // is small enough; + 1 is for the remainder
-        for _ in 0..self.options.num_fri_layers(evaluations.len()) + 1 {
+        for _ in 0..self.options.num_fri_layers(evaluations.len()) {
```

Comment:
The comment above this line needs to be updated (I think we can just remove the "+1 ... " part of the comment.

Reply:
Yes, thank you!